src/screens/SubjectsIndex/SubjectsPageContainer.js

```diff
@@ -7,8 +7,21 @@ import { EDIT_PATH } from 'paths'
 import ResourcesTable from '../../components/ResourcesTable'
 import { columns } from './table'
 
+function slicePages(page, totalPages) {
+  const allPages = Array.from(Array(totalPages).keys())
+  let leftPage = page - 2 < 0 ? 0 : page - 2
+  let rightPage = leftPage + 5
+  if (totalPages > 5 && rightPage > totalPages - 1) {
+    leftPage = totalPages - 5
+    rightPage = totalPages
+  }
```

Comment:
[minor] You might want to make some of these magic numbers into constants, e.g. `const MAX_NUMBER_OF_PAGES_SHOWN_IN_PAGING = 5`